tests/eclient/image/files/entrypoint.sh

```diff
@@ -19,8 +19,23 @@ rm -rf /run/*
 mkdir -p /run/sshd
 mkdir -p /run/nginx
 
+# eclient can be also used as a router app
+IP_FORWARDING="$(sysctl -n net.ipv4.ip_forward)"
+if [ "$IP_FORWARDING" -eq 1 ]; then
+    # With bare containers (eve.accel=false), IP forwarding is always enabled.
+    echo "IP forwarding is already enabled"
+else
+    echo "Enabling IP forwarding..."
+    sysctl -w net.ipv4.ip_forward=1 && echo "IP forwarding is now enabled"
+fi
+
 nginx
 
 /usr/sbin/sshd -h /root/.ssh/id_rsa
 
-avahi-daemon
+avahi-daemon -D
```

Comment:
Potentially we will hist the problem with [app logs test](https://github.com/lf-edge/eden/blob/03270353b43750110fa13746ae6c2142ca8aeb8d/tests/eclient/testdata/app_logs.txt#L27). WDYT, probably we should put some comment on top, e.g. `starting eclient` and wait for it in test? Potentially we may check for `IP forwarding` after your changes...

Reply:
Added `echo "Started eclient"`. Tested locally and confirmed that the `app_log` test will pass with these changes:
```
diff --git a/tests/eclient/testdata/app_logs.txt b/tests/eclient/testdata/app_logs.txt
index 7aff470..dc91753 100644
--- a/tests/eclient/testdata/app_logs.txt
+++ b/tests/eclient/testdata/app_logs.txt
@@ -9,8 +9,8 @@ test eden.app.test -test.v -timewait 15m RUNNING eclient &
 
 # We check for pattern exists in {{template "eclient_image"}}
 message 'Waiting for App logs'
-test eden.lim.test -test.v -timewait 15m -test.run TestAppLog -app eclient -out content 'content:.*ubuntu-http.*'
-stdout 'ubuntu-http-server'
+test eden.lim.test -test.v -timewait 15m -test.run TestAppLog -app eclient -out content 'content:.*Started.eclient.*'
+stdout 'Started eclient'
 
 wait
 
@@ -24,8 +24,8 @@ test eden.app.test -test.v -timewait 5m RUNNING eclient &
 
 # We check for pattern exists in {{template "eclient_image"}}
 message 'Waiting for App logs after app restart'
-test eden.lim.test -test.v -timewait 5m -test.run TestAppLog -app eclient -out content 'content:.*ubuntu-http.*'
-stdout 'ubuntu-http-server'
+test eden.lim.test -test.v -timewait 5m -test.run TestAppLog -app eclient -out content 'content:.*Started.eclient.*'
+stdout 'Started eclient'
 
 wait
```